### PR TITLE
Change mobile view for user manager page

### PR DESCRIPTION
**Pull request by ekanshgupta123**
Screen Shot 2022-07-16 at 2 44 30 PM

### CODE REVIEW

File: src/Components/Frame/HeroFrame.css [hunk 1]
```diff
@@ -61,7 +61,7 @@
 
 @media screen and (max-width:400px) {
 	.name {
-		display: none !important
+		/* display: none !important; */
```
Comment (evanugarte) [inline src/Components/Frame/HeroFrame.css:64]: if u dont need this then just remove the rule entirely

File: src/Pages/Overview/Overview.css [hunk 2]
```diff
@@ -129,3 +144,34 @@ thead.td {
...
+    margin-left: 20%;
+  }
+}
+
+@media screen and (min-width: 700px) and (max-width: 1280px) {
+  .layout{
+    padding-top: 70px;
+    height: 100%;
+  }
+}
```
Comment (evanugarte) [inline src/Pages/Overview/Overview.css:177]:
```suggestion
}

```

**Review by evanugarte — APPROVED** [review]
ez

**Review by neelp03 — CHANGES_REQUESTED** [review]
What is you also add the same amount of space thats on the left side of the screen to the right side? So it looks centered.

**Review by neelp03 — APPROVED** [review]

**Pull request merged**